### ISSUE TITLE
📣 Elementor 3.26: Important Updates for Developers

### ISSUE THREAD

**Issue by nicholaszein** [elementor/elementor#29436]
Hello Developers! 👋 

As we prepare for the upcoming release of **Elementor 3.26**, currently in beta testing, we want to ensure you're informed about critical changes that may impact your work.

> **_If you are an Elementor Addon developer, have created custom widgets for your projects, or use custom code or CSS in any capacity, please review the following updates and advisories to prepare your projects accordingly._**

### ⚠️ Developer Advisory: Breaking Changes to prepare for
- **Scheme Method Deletion:** The outdated **Scheme** method is being removed. Replace its usage with the updated **Globals** method to ensure compatibility.
- **Migration to Swiper 8:** The Swiper library has been upgraded from version 5 to 8. Update your scripts and customizations to align with this new version.
- **Deprecated Widget Container CSS class:** The `.elementor-widget-container` class is now deprecated. Update any styles or scripts relying on this class to avoid issues.
- **Add Support for Optimized DOM:** Significant DOM improvements have been made. Ensure your custom widgets or styling are compatible with the streamlined output.

### ⬆️ Updates to Experiments and Features
- **Optimized Control Loading merged to Core:** Control loading enhancements are now standard, improving the editor's performance.
- **Upgrade Swiper Library merged to Core:** The Swiper 8 upgrade is finalized and integrated into the core for better performance and features.
- **Nested Elements upgraded to Stable:** Nested Elements, previously experimental, is now a stable feature, enabling more advanced designs.
- **Element Caching active by default on existing websites:** This optimization improves loading speeds by caching static content automatically.

### 🔑 Other Key Areas to be aware of
- **Updated Browser Support:** Elementor 3.26 drops support for Safari versions earlier than 15.5, aligning with modern standards.
- **Optimized DOM size for Carousels:** The structure of carousels has been refined to reduce DOM size and improve performance.
- **Optimized DOM size for other widgets:** Additional widgets now benefit from reduced DOM size, further enhancing site efficiency.

### ℹ️ For complete details, check out the full developer update: [Elementor 3.26 Developer Update](https://developers.elementor.com/elementor-3-26-developers-update/).

> _We encourage you to test these changes in your development environments and make the necessary adaptations to ensure compatibility going forward._

Elementor version 3.26 is scheduled to be released on **December 16th, 2024** (estimated).

Join the Discussion about the version 3.26 Beta here: [https://github.com/orgs/elementor/discussions/29403](https://github.com/orgs/elementor/discussions/29403)

----

Make sure you have read our **Developers Documentation**, followed our **Best Practices** guide, our **Recent Deprecations** list, and subscribed to our **Developers Newsletter**:

- **📚 Developers Center:** Learn how to leverage the wide-range of tools and options Elementor has to offer.
**[https://developers.elementor.com](https://developers.elementor.com)**

- **🛠️ Best Practices:** Follow our Best Practices Guide to align your addons with Elementor's standards and optimize performance.
**[https://developers.elementor.com/docs/getting-started/best-practices/](https://developers.elementor.com/docs/getting-started/best-practices/)**
- **🚨 Recent Deprecations:** Stay informed about Recent Deprecations to update your code accordingly and maintain compatibility.
**[https://developers.elementor.com/recent-deprecations/](https://developers.elementor.com/recent-deprecations/)**
- **📬 Developer Updates:** Subscribe to the Developers Newsletter to receive the latest updates, feature releases, and important announcements directly in your inbox.
**[https://elementor.com/pages/addon-developers-corner/](https://elementor.com/pages/addon-developers-corner/)**

**Comment by BluePointDigital** [comment]
I agree. I just had to rollback. It broke SO MANY separate features on a very large site it was just ridiculous to try and fix them.

I've just about had it with Wordpress and this ecosystem. Just started my next project with Sveltekit and am taking the dive at learning a new platform.
